example.js

```diff
@@ -0,0 +1,25 @@
+
+var Stagger = require('./lib/stagger');
+
+var stagger = Stagger.getDefault({
+  "pid": "1234",
+  "cmd": "./hello"
```

Comment:
Yes if they create their own instance of the stagger.Client. I prefer libraries who don't provide that default singleton but we're kind of stuck with it for the moment.